zjit/src/codegen.rs

```diff
@@ -464,8 +464,15 @@ fn gen_send_without_block(
     self_val: &InsnId,
     args: &Vec<InsnId>,
 ) -> Option<lir::Opnd> {
-    // Spill the receiver and the arguments onto the stack. They need to be marked by GC and may be caller-saved registers.
+    // Spill locals onto the stack.
```

Comment:
```suggestion
    // Spill locals onto the stack.
    // TODO: Don't spill locals eagerly; lazily reify frames
```

Reply:
thanks, I forgot to leave a TODO for this one https://github.com/ruby/ruby/pull/13579/commits/829a7314f1033fc03c2e52990a8370295e1309ab